tests/release/pipelines/README.md

```diff
@@ -1,3 +1,13 @@
-# Release Bundles Tests
+# Release Pipelines Tests
 
-This suite contains e2e-tests for testing release pipelines from repository [release-service-catalog](https://github.com/redhat-appstudio/release-service-catalog/tree/main), those tests run against RHTAP RH-Stage.
\ No newline at end of file
+This suite contains e2e-tests for testing release pipelines from repository [release-service-catalog](https://github.com/redhat-appstudio/release-service-catalog/tree/main), those tests run against RHTAP RH-Stage, the tests running on two dedicated tenant namespaces on RH Stage `dev-release-team-tenant` and `managed-release-team-tenant`.
+
+## All tests should have the label `release-pipelines` to avoid it be running within the CI e2e-test 
+
+## prerequisites: 
+    
+   **required the below environment variables defined:** 
+
+    	- RHTAP_TOOLCHAIN_API_URL: Tool Chain API host to connect the RH Stage
+		- RHTAP_KEYLOAK_URL:       SSO Url to authenticate the connection. 
+		- RHTAP_OFFLINE_TOKEN :    This should contain the offline token to connect RHTAP RH Stage
```

Comment:
I think apart from these, if people want to run these tests, they have to have access to dev/managed workspaces, right.

I would mention here that it is a prerequisite and that they have to ask the team to get a permission (I wouldn't be too specific about the exact team name here in the public repo).

Reply:
I've added this point as prerequisite